app/src/main/java/com/example/groceriesmanager/Fragments/UserProfileFragment.java

```diff
@@ -88,40 +92,55 @@ public void done(ParseException e) {
         rlMyRecipes.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
+                String uri;
                 if (rvMyRecipes.getVisibility() == View.GONE){
                     rvMyRecipes.setVisibility(View.VISIBLE);
-                    // todo: set the image resource
+                    uri = "@drawable/collapse_arrow";  // where myresource (without the extension) is the file
                 }
                 else{
                     rvMyRecipes.setVisibility(View.GONE);
-                    // todo: set the image resource
+                    uri = "@drawable/expand_arrow";  // where myresource (without the extension) is the file
                 }
+                // set image resource
```

Comment:
These 3 lines show up several times for getting a drawable resource and then setting the the image on an element.   Perhaps you can pull these lines out into a helper function and simplify the way this is done so there isn't so much duplicate code here.